miden/examples/hashing/blake3/blake3.masm

```diff
@@ -0,0 +1,15 @@
+use.std::crypto::hashes::blake3
+
+begin
+    # hash_2to1 pops top 16 elements (64 bytes) from the stack, compute their hash and puts the 
+    # resulting 8 elements (32 bytes) back to the stack.
+    exec.blake3::hash_2to1
+
+    # check the correctness of the hashing result by comparing it with precomputed correct values
+    push.2966802291.217477570.4255647887.29124900.290943167.831231409.2159343350.4191074185
```

Comment:
I would use `push.0x` notation here as this is how hashes are usually represented.

Reply:
Often hashes are represented as "long string" hex, but in our case, this form will consist of half of zeros. Is it OK to use short form? So use this
```
push.0xB0D5CF73.0x0CF671C2.0xFDA8088F.0x01BC6924
```
instead of this
```
push.0x73CFD5B000000000C271F60C000000008F08A8FD000000002469BC0100000000
```

Reply:
> In short from they will look like so:
> 
> ```
> push.0xB0D5CF73.0x0CF671C2.0xFDA8088F.0x01BC6924
> push.0x115770BF.0x318B95B1.0x80B4F6F6.0xF9CEB789
> ```

Let's go with this approach.

Reply:
And in this case, we should be able to do something like:
```
push.0xB0D5CF73.0x0CF671C2.0xFDA8088F.0x01BC6924.0x115770BF.0x318B95B1.0x80B4F6F6.0xF9CEB789
```